test/unit/Helper/GMUObectiveCTestHelper.m

```diff
@@ -0,0 +1,31 @@
+/* Copyright (c) 2020 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#import "GMUObectiveCTestHelper.h"
+
+@implementation GMUObectiveCTestHelper
+
++ (BOOL)catchObjectiveCException:(void(^)(void))tryBlock error:(__autoreleasing NSError **)error {
```

Comment:
It's unfortunate that this is needed since an `NSException` is thrown in our APIs. I'm going to file an issue so that we can modify the API to accept an `NSError` instead to improve Swift/Objective-C interop

Reply:
Filed in #315

Reply:
Yeah, have struggled a lot for this :(